core/templates/pages/exploration-editor-page/exploration-editor-page.component.html

```diff
@@ -50,6 +50,7 @@ <h1>{{ getNavbarText() }}</h1>
            placement="bottom"
            tabindex="0"
            aria-label="Exploration Translation Button"
+           role="button"
```

Comment:
Is there any specific reason for not adding the `role="button"` to one `anchor` tag on line 17 but using it in all other places in this file?

Reply:
@DubeySandeep Thanks for pointing! I was running lighthouse test, that did not give error related to that anchor tag, because it was in selection. But it is good to have consistent semantics for all buttons, so  I have `role="button"` there also.